<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Ankush" ignored="0" name="Test" tests="9" failures="2" timestamp="04 Mar 2024 16:29:24 GMT" time="0.289" errors="0">
  <testcase name="good" time="0.059" classname="Test.day4">
    <skipped/>
  </testcase> <!-- good -->
  <testcase name="good" time="0.005" classname="Test.day4">
    <skipped/>
  </testcase> <!-- good -->
  <testcase name="good" time="0.003" classname="Test.day4">
    <skipped/>
  </testcase> <!-- good -->
  <testcase name="good" time="0.006" classname="Test.day4">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.assertTrue(Assert.java:20)
	at junit.framework.Assert.assertTrue(Assert.java:27)
	at Test.day4.good(day4.java:21)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- good -->
  <testcase name="goods" time="0.003" classname="Test.day4"/>
  <testcase name="hey" time="0.011" classname="Test.day4">
    <skipped/>
  </testcase> <!-- hey -->
  <testcase name="hey" time="0.003" classname="Test.day4">
    <skipped/>
  </testcase> <!-- hey -->
  <testcase name="hey" time="0.009" classname="Test.day4">
    <skipped/>
  </testcase> <!-- hey -->
  <testcase name="hey" time="0.003" classname="Test.day4">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.assertTrue(Assert.java:20)
	at junit.framework.Assert.assertTrue(Assert.java:27)
	at Test.day4.hey(day4.java:14)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- hey -->
</testsuite> <!-- Test -->
